src/engine/renderer/gl_shader.h

```diff
@@ -2194,6 +2194,26 @@ class u_Lights :
 	}
 };
 
+class GLShader_generic2D :
+	public GLShader,
```

Comment:
Could use a comment/TODO explaining how a lot of this stuff is copy-pasted junk which is not actually used by the shader.

Reply:
Done.